seed/urls/main.py

```diff
@@ -151,6 +153,43 @@
         name='delete_organization_buildings'
     ),
 
+    # timeseries data
```

Comment:
yes, agree that dropping "url" from the endpoint names is good.

why is retrieve_monthly_data listed twice?

I think this gets to the big question about how RESTful we want to make the APIs now.
These are mostly just doing CRUD with the time-series resource, but the parameters are encoded (partially) in the endpoint name. 
Are there issues with POSTing a big time series file to an endpoint? I thought there was some limit to the size of the data that can be sent through a POST command.

We're going to have to clean up all the endpoints at some point, so not the end of the world if we wait, but good not to start with a design we know we're going to change.